src/components/webview/IntuitaPanel.ts

```diff
@@ -89,7 +125,41 @@ export class IntuitaPanel {
 	}
 
 	public render() {
-		this.__panel?.reveal();
+		const initWebviewPromise = new Promise((resolve, reject) => {
+			this.__panel?.reveal();
+
+			const timeout = setTimeout(() => {
+				this.__panel?.dispose();
```

Comment:
is it a promise?

Reply:
panel.reveal is sync method, but it takes some time to mount react application (and subscribe to messages) after its revealed. And because of that there can be glitches where webview ignores some initial events. So i had to wrap it in a promise. I will rewrite this method if i find better solution.